src/components/homeTabs/GIVstream.tsx

```diff
@@ -238,6 +237,7 @@ export const TabGIVstreamBottom = () => {
 						</GsDataBlock>
 					</Col>
 				</Row>
+				<GIVstreamHistory />
```

Comment:
what about history title? shouldn't it be moved too?

Reply:
Done!